README.md

```diff
@@ -178,6 +181,45 @@ docker-compose up -d op-geth-replica op-node-replica op-blockscout-replica
 
 Open the replica explorer ([http://127.0.0.1:4002/](http://127.0.0.1:4002/)). L2 block is synchronized with about a 1-minute delay. Synchronization can be sped up by reducing the `--verifier.l1-confs` in the op-node-replica service.
 
+## Setup Message Relayer
+
+1. Go to message_relayer folder
+```shell
+cd message_relayer
+```
+2. Set up env 
+
+```shell
+cp .env.example .env
+
+# Edit the .env file to set your environment variables:
+vi .env
+```
+To get MESSAGE_RELAYER__PROVER_PRIVATE_KEY, MESSAGE_RELAYER__FINALIZER_PRIVATE_KEY go to env from private-opstack
+
+To get MESSAGE_RELAYER__ADDRESS_MANAGER, MESSAGE_RELAYER__L1_CROSS_DOMAIN_MESSENGER,
+MESSAGE_RELAYER__PORTAL_ADDRESS,
+MESSAGE_RELAYER__OUTPUT_ORACLE
+
+```sh
+jq -r .AddressManager tmp/oasys/L1/build/Build.s.sol/latest/addresses.json
+jq -r .L1CrossDomainMessengerProxy tmp/oasys/L1/build/Build.s.sol/latest/addresses.json
+jq -r .OptimismPortalProxy tmp/oasys/L1/build/Build.s.sol/latest/addresses.json
+jq -r .L2OutputOracleProxy tmp/oasys/L1/build/Build.s.sol/latest/addresses.json
+```
+
+
+3. Deploy Multisig
```

Comment:
I noticed that the multisig2 is pre deployed no L1 at the `0x5200000000000000000000000000000000000022`
so, don't have to deploy it again.
please ommit this step.